vendor/bat-native-ledger/src/bat_helper.cc

```diff
@@ -2618,6 +2619,24 @@ uint64_t getMediaDuration(const std::map<std::string, std::string>& data,
   return duration;
 }
 
+bool IsValidLeadingUrlSubString(
+    const std::string& url_to_validate,
+    const std::string& to_match) {
+  GURL gurl(url_to_validate);
+  GURL gurl_to_match(to_match);
+  if (gurl_to_match.is_valid() && gurl.is_valid() &&
+      gurl_to_match.host() == gurl.host()) {
```

Comment:
if you're trying to match subdomains you can use `DomainIs`

Reply:
Done.